tasks/git/src/main/java/com/walmartlabs/concord/plugins/git/v2/GithubTaskV2.java

```diff
@@ -34,17 +35,28 @@ public class GithubTaskV2 implements Task {
 
     private final GitHubTask delegate;
 
-    private final Map<String, Object> defaults;
+    private final Map<String, Object> policyDefaults;
+
+    private final GitSecretService secretService;
 
     @Inject
-    public GithubTaskV2(Context ctx) {
-        this.defaults = ctx.defaultVariables().toMap();
+    public GithubTaskV2(Context ctx, SecretService secretService) {
+        this.policyDefaults = ctx.defaultVariables().toMap();
         this.delegate = new GitHubTask(ctx.processConfiguration().dryRun());
+        this.secretService = new SecretServiceV2(ctx.secretService());
     }
 
     @Override
     public TaskResult execute(Variables input) {
```

Comment:
Can we add https://github.com/walmartlabs/concord/blob/master/runtime/v2/sdk/src/main/java/com/walmartlabs/concord/runtime/v2/sdk/SensitiveData.java here?

Reply:
yep yep yep 👍